validator/test/fixtures/assets/bigquery_cmek/data.json

```diff
@@ -0,0 +1,157 @@
+                    "fields": [
+                        {
+                            "description": "",
+                            "name": "string_field_0",
+                            "type": "TYPE_STRING"
+                        }
+                    ]
+                },
+                "tableReference": {
+                    "datasetId": "anand_bq_test_3",
+                    "projectId": "anand-spanner",
+                    "tableId": "test_google_encryption_key"
+                }
+            }
+        }
+    },
+    {
+        "name": "//cloudkms.googleapis.com/projects/anand-spanner/locations/global/keyRings/test/cryptoKeys/test-1/cryptoKeyVersions/1",
+        "asset_type": "cloudkms.googleapis.com/CryptoKeyVersion",
+        "resource": {
+            "version": "v1",
+            "discovery_document_uri": "https://cloudkms.googleapis.com/$discovery/rest",
+            "discovery_name": "CryptoKeyVersion",
+            "data": {
+                "algorithm": "GOOGLE_SYMMETRIC_ENCRYPTION",
+                "createTime": "2019-05-15T17:01:01.882806557Z",
+                "generateTime": "2019-05-15T17:01:01.882806557Z",
+                "name": "projects/anand-spanner/locations/global/keyRings/test/cryptoKeys/test-1/cryptoKeyVersions/1",
+                "protectionLevel": "SOFTWARE",
+                "state": "ENABLED"
+            }
+        }
+    },
+    {
+        "name": "//cloudkms.googleapis.com/projects/anand-spanner/locations/global/keyRings/test/cryptoKeys/test-1",
+        "asset_type": "cloudkms.googleapis.com/CryptoKey",
+        "resource": {
+            "version": "v1",
+            "discovery_document_uri": "https://cloudkms.googleapis.com/$discovery/rest",
+            "discovery_name": "CryptoKey",
+            "parent": "//cloudkms.googleapis.com/projects/anand-spanner/locations/global/keyRings/test",
+            "data": {
+                "createTime": "2019-05-15T17:01:01.882806557Z",
+                "name": "projects/anand-spanner/locations/global/keyRings/test/cryptoKeys/test-1",
+                "nextRotationTime": "2019-08-13T05:00:00Z",
+                "primary": {
+                    "algorithm": "GOOGLE_SYMMETRIC_ENCRYPTION",
+                    "createTime": "2019-05-15T17:01:01.882806557Z",
+                    "generateTime": "2019-05-15T17:01:01.882806557Z",
+                    "name": "projects/anand-spanner/locations/global/keyRings/test/cryptoKeys/test-1/cryptoKeyVersions/1",
+                    "protectionLevel": "SOFTWARE",
+                    "state": "ENABLED"
+                },
+                "purpose": "ENCRYPT_DECRYPT",
+                "rotationPeriod": "7776000s",
+                "versionTemplate": {
+                    "algorithm": "GOOGLE_SYMMETRIC_ENCRYPTION",
+                    "protectionLevel": "SOFTWARE"
+                }
+            }
+        }
+    },
+    {
+        "name": "//spanner.googleapis.com/projects/anand-spanner/instances/demo",
```

Comment:
let's delete the spanner data here since they seem unrelated?

Reply:
Fixed.